GlossaryOfTerms.ttl

```diff
@@ -5257,3 +5263,76 @@ owl:IrreflexiveProperty
                             <https://twitter.com/hashtag/SSEO#this> ,
                             <https://twitter.com/hashtag/ContentManagement#this> ;
     skos:inScheme           :this .
+
+<http://data.openlinksw.com/oplweb/glossary-term/EnterpriseKnowledgeGraph#this>,
+    a                       skos:Concept ;
+    wdrs:describedby        source: ;
+    schema:name             "Enterprise Knowledge Graph" ;
+    schema:description      """An Enterprise Knowledge Graph (EKG) is a Knowledge Graph that springs from and
+                               centers on a particular Enterprise, typically a Company, most often building 
+                               outward from the Enterprise "Home Page" on the WorldWideWeb. Connections to
+                               Suppliers, Customers, Employees, Products, Competitors, and Partners are just
+                               a sampling of what may be found in an EKG.
+                            """ ;
+    schema:subjectOf        <http://www-kasm.nii.ac.jp/iswc2016/papers/paper_A29_.pdf> ,
+                            <https://www2.deloitte.com/content/dam/Deloitte/de/Documents/operations/knowledge-graphs-pov.pdf> ,
+                            <https://dmccreary.medium.com/enterprise-knowledge-graph-2020-58fc0c494f7c> ,
+                            <https://enterprise-knowledge.com/what-is-an-enterprise-knowledge-graph-and-why-do-i-want-one/> ;
+    skos:broader            <http://data.openlinksw.com/oplweb/glossary-term/KnowledgeGraph#this> ;
+    skos:related            <http://data.openlinksw.com/oplweb/glossary-term/ProductKnowledgeGraph#this> ,
+                            <http://data.openlinksw.com/oplweb/glossary-term/ProfileKnowledgeGraph#this> ,
+                            <http://data.openlinksw.com/oplweb/glossary-term/DataMesh#this> ,
+                            <https://dbpedia.org/resource/Enterprise_social_graph>;
+    skos:inScheme           :this .
+
+<http://data.openlinksw.com/oplweb/glossary-term/ProductKnowledgeGraph#this>,
+    a                       skos:Concept ;
+    wdrs:describedby        source: ;
+    schema:name             "Product Knowledge Graph" ;
+    schema:description      """A Product Knowledge Graph is a Knowledge Graph that springs from and
+                               centers on a particular Product, most often building outward from the 
+                               Product "Home Page" on the WorldWideWeb. Connections to Users, Competing
+                               Products, Complementary Products, and Ingredients (physical and otherwise)
+                               are just a sampling of what may be found in a Product Knowledge Graph.
```

Comment:
```suggestion
    schema:description      """A Product Knowledge Graph is a Knowledge Graph derived from entity
                               relationships across various product-associated domains, such as 
                               Users, Producers, Competing Products, Complementary Products, 
                               Components/Ingredients (physical and otherwise), and many others.
```